createdisk.sh

```diff
@@ -199,6 +216,19 @@ kernel_cmd_line=$(${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'cat /proc/cm
 # SCP the vmlinuz/initramfs from VM to Host in provided folder.
 ${SCP} core@api.${CRC_VM_NAME}.${BASE_DOMAIN}:/boot/ostree/rhcos-${ostree_hash}/* $1
 
+# Download the hyperV daemons dependency on host
+mkdir $1/hyperv
+sudo yum install --downloadonly --downloaddir $1/hyperv hyperv-daemons
+
+# SCP the downloaded rpms to VM
+${SCP} -r $1/hyperv core@api.${CRC_VM_NAME}.${BASE_DOMAIN}:/home/core/
+
+# Install the hyperV rpms to VM
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rpm-ostree install /home/core/hyperv/*.rpm'
+
+# Remove the packages from VM
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- rm -fr /home/core/hyperv
```

Comment:
:+1

Reply:
This means we will have the hyperv-daemons on libvirt, vbox and hyperkit too? Probably not a huge deal, but maybe we'll want to avoid these in the future.